src/WFS.js

```diff
@@ -39,8 +40,15 @@ L.WFS = L.FeatureGroup.extend({
 
     var that = this;
     this.describeFeatureType(function () {
-      if (that.options.showExisting) {
-        that.loadFeatures();
+      if (that.options.bbox instanceof L.LatLngBounds) {
```

Comment:
Here filter should be applied only if set showExisting flag. And using filter option as I say in previous comment.
```javascript
if (that.options.showExisting) {
   that.loadFeatures(that.options.filter);
}
```
filter should not be instantiated in plugin core - it must do code of application.